settings.py

```diff
@@ -486,3 +486,6 @@
 else:
     SOCIAL_AUTH_PIPELINE = SOCIAL_AUTH_PIPELINE_BASE + SOCIAL_AUTH_PIPELINE_USER_EXIST + \
                            SOCIAL_AUTH_PIPELINE_ASSOCIATE_USER + SOCIAL_AUTH_PIPELINE_LOG
+
+#S3 specific settings
+S3_ARN_ASSUME_ROLE = os.environ.get('S3_ARN_ASSUME_ROLE', 'none')
```

Comment:
Using a magic string for `'none'` is not great practice as it is really confusing for developers. It would be better if this did not have a default value, and then if there are cases where you actually need this magic string value in calls to s3 you would use something like `S3_ARN_ASSUME_ROLE  or 'none'` to make it clear that its intentional